app/controllers/placements/providers/partner_schools/placements_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Placements::Providers::PartnerSchools::PlacementsController < ApplicationController
+  before_action :set_provider, only: %i[index show]
+  before_action :set_partner_school, only: %i[index show]
+  def index
+    placements_assigned_to_provider = @partner_school.placements.where(provider_id: @provider.id)
+    other_placements = @partner_school.placements.where.not(id: placements_assigned_to_provider.ids)
+    @placements_assigned_to_provider = placements_assigned_to_provider.decorate
+    @other_placements = other_placements.decorate
+  end
+
+  def show
+    @placement = @partner_school.placements.find(params.require(:id)).decorate
+    @partner_school = @partner_school.decorate
```

Comment:
With the `decorates_association :school` in the `PlacementDecorator` you can do:
```suggestion
    @partner_school = @placement.school
```